### ISSUE TITLE
License Change from AGPL-3.0

### ISSUE THREAD

**Issue by a5m0** [certsocietegenerale/FIR#206]
### Request Type
Feature Request

### Problem Description
Would you consider changing your license from AGPL-3.0? This license prevents use by many organizations that would like to use and contribute to this project.

### Possible Solutions
Apache 2.0
MIT License

**Comment by tomchop** [comment]
The [license](https://github.com/certsocietegenerale/FIR/blob/master/LICENSE) is not **A**GPL but GPL. There's a substantial difference!

**Comment by a5m0** [comment]
My apologies I meant GPL, would you be able to make it Apache 2.0 like yeti-platform?

**Comment by tomchop** [comment]
I will let the other project members pitch in (CERT Société Générale owns the project so changing the license might take a while, if it can be done at all). 

That said, as far as I know GPL3 does allow commercial use of the tool; IANAL but most orgs should be able to use this without any problems.

![image](https://user-images.githubusercontent.com/1257972/29418596-14ef6238-836d-11e7-998e-77986f6783ea.png)

**Comment by a5m0** [comment]
My organization just approved me to contribute to Yeti because of the Apache 2 license but not FIR because it's not Apache 2.0 or MIT License. I do not know how many other orgs have issues with these licenses, but it's possible I'm not alone.

**Comment by tomchop** [comment]
Fair enough. I have no problem in changing the license, but I need all other contributors to agree. @certsocietegenerale, what do you think?
